manifests/config.pp

```diff
@@ -49,9 +49,11 @@
     unit     => "${foreman::foreman_service}.service",
     content  => template('foreman/foreman.service-overrides.erb'),
   }
-
-  file { $foreman::app_root:
-    ensure  => directory,
+  
```

Comment:
Looks like there's trailing whitespace on this line which fails puppet-lint.

Reply:
oh, you're right. should not be corrected :)